Applications/MaterialManager/Contracts/Processing/Optimization/BookheightParameterSet.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using HomagConnect.Base.Contracts.Enumerations;
+using HomagConnect.Base.Contracts.Interfaces;
+using Newtonsoft.Json;
+
+namespace HomagConnect.MaterialManager.Contracts.Processing.Optimization;
+
+public class BookheightParameterSet : IValidatableObject, IContainsUnitSystemDependentProperties
+{
+    private const int _MaterialGroupNameMaxLength = 50;
+
+    /// <summary>
+    ///     Gets or sets the name of the material group.
```

Comment:
Please remove all of these empty spaces in front. (same for the other properties)

Reply:
removed the spaces. resolved